tests/tiles/tiles.spec.ts

```diff
@@ -601,6 +601,24 @@ const goodMinimalBoundingTileTests = [
     boundingBox: new BoundingBox(160, -5, 170, 5),
     expectedTile: new Tile(1, 0, 0, 1),
   },
+  {
+    testCaseName: 'for non-default metatile (2)',
+    boundingBox: new BoundingBox(5, 5, 35, 35),
+    metatile: 2,
+    expectedTile: new Tile(4, 1, 3, 2),
+  },
+  {
+    testCaseName: 'for non-default metatile (2) with a large bounding box intersecting tile grid at zoom 0',
+    boundingBox: new BoundingBox(-180, -90, 180, 90),
+    metatile: 2,
+    expectedTile: new Tile(0, 0, 0, 2),
```

Comment:
Is it even possible to have a `metatile` at zoom 0? Or am I missing something?

Reply:
a single `metatile` will span both tiles in zoom 0. it will lack the tiles in the `y` direction. they may considered as "virtual"